src/validators/typed_dict.rs

```diff
@@ -222,11 +264,21 @@ impl Validator for TypedDictValidator {
                                     fs.push(field.name_pystring.clone_ref(py));
                                 }
                             }
-                            Err(ValError::LineErrors(line_errors)) => {
-                                for err in line_errors {
-                                    errors.push(err.with_outer_location(field.name.clone().into()));
+                            Err(ValError::LineErrors(line_errors)) => match field.on_error {
+                                OnError::Raise => {
+                                    for err in line_errors {
+                                        errors.push(err.with_outer_location(field.name.clone().into()));
+                                    }
                                 }
-                            }
+                                OnError::Omit => continue,
+                                OnError::FallbackOnDefault => {
+                                    if let Some(ref default) = field.default {
```

Comment:
perhaps we should add

```rs
impl TypedDictField {
    fn default_value(&self, ...) -> Option<PyOjbect> {
        // this logic
    }
}
```
Then we can use it here and below.

Reply:
Good idea